doc/source/reference/random/index.rst

```diff
@@ -18,41 +85,36 @@ to use those sequences to sample from different statistical distributions:
   distribution (such as uniform, Normal or Binomial) within a specified
   interval.
 
-Since Numpy version 1.17.0 the Generator can be initialized with a
-number of different BitGenerators. It exposes many different probability
-distributions. See `NEP 19 <https://www.numpy.org/neps/
-nep-0019-rng-policy.html>`_ for context on the updated random Numpy number
-routines. The legacy `RandomState` random number routines are still
-available, but limited to a single BitGenerator. See :ref:`new-or-different` 
+In short, BitGenerators are the parts responsible for the randomization
+and the state, whereas Generators are responsible for utilizing these
+randomized inputs.
```

Comment:
I disagree, I think it's a useful summary, as the previous explanation is more precise but less accessible.

Reply:
>I disagree, I think it's a useful summary, as the previous explanation is more precise but less accessible.

Good point. I looked at the file after it was formatted, and it flowed a lot better than I thought it would when I was looking at the unformatted text. @ilayn, disregard this suggestion.

Reply:
Hey @MatteoRaso, while I appreciate you jumping in and helping to review PRs, your tone and review style isn't really okay I'm afraid. There are a couple of issues I see:

1. Something like _"`<username>`, disregard this suggestion."_ sounds like an order (which is bad). You could have inserted the word "please" instead, or just left it out completely and hit the "Resolve conversation" button.
2. Starting reviewing with detailed grammatical points and suggested rephrases is an anti-pattern. Please try to focus on the big picture first. In case of this PR, that's how the information is structured, and whether the examples and points made are clear/helpful.

I've noticed this on a few other PRs as well. That includes your own PRs - _"Any updates to this PR?"_ is not how to ask for attention from very busy reviewers, I suggest trying something like _"It's been 10 days and I believe I have addressed all previous comments, can you please have another look at this PR?"_. 

Can you please read https://numpy.org/devdocs/dev/reviewer_guidelines.html#communication-guidelines, try to phrase things in a more kind/empathetic fashion, and avoid nitpicking and grammar?

Reply:
@rgommers Sorry about that. I do try to be polite, but it's difficult for me to convey tone online. Yes, I'll be sure to read the reviewer guideline and carefully apply it in future reviews. @ilayn sorry if I came off as rude in this PR.

Reply:
thanks, much appreciated!

Reply:
I didn't quite take any offense but I can see how it can be taken the other way but regardless thanks all for the quick resolution. Hopefully, this will improve the overall docs for everyone.